SentryPlugin.php

```diff
@@ -112,5 +129,50 @@ public function init()
         craft()->onError = function ($event) use ($client) {
             $client->captureMessage($event->message);
         };
+
+        return $this;
+    }
+
+
+    /**
+     * Includes the JS for front-end sentry.
+     * @return $this
+     */
+    protected function configureFrontendSentry()
+    {
+        // Get plugin settings
+        $settings = $this->getSettings();
+
+        if (!$this->isWebRequest()) {
+            return $this;
+        }
+
+        if (!$settings->reportJsErrors) {
+            return $this;
+        }
+
+        craft()->templates->includeJsFile('https://cdn.ravenjs.com/1.1.22/jquery,native/raven.min.js');
```

Comment:
@benjolabs : is including what looks like another version of jQuery going to mess with any of our FE stuff?